### PR TITLE
Models Created!

### PR DESCRIPTION
**Pull request by abhijithanilkumar**
Models for the website are added in the new 'website' app. @rohitvarkey Have a look at this !

### CODE REVIEW

File: src/website/models.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+from __future__ import unicode_literals
+
+from django.db import models
+from django.core.validators import RegexValidator
+from django_markdown.models import MarkdownField
+
+class News(models.Model):
+    choice = [('C', 'Campus News'),
```
Comment (rohitvarkey) [inline src/website/models.py]: category?

File: src/website/models.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    name = models.CharField(max_length=50)
+    prof_pic = models.ImageField(upload_to='author_profile_pics/%Y-%m-%d/',null=True,blank=True)
+
+    def __str__(self):
+        return self.name
+
+class Articles(models.Model):
+    title = models.CharField(max_length=200)
+    author = models.ForeignKey(Author)
+    details = MarkdownField()
```
Comment (rohitvarkey) [inline src/website/models.py]: content?

**Comment by abhijithanilkumar** [comment]
@rohitvarkey Have a look at this please.

**Comment by rohitvarkey** [comment]
Looks good.

**Pull request merged**